rules/integrations/aws/impact_cloudwatch_log_stream_deletion.toml

```diff
@@ -23,7 +23,65 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS CloudWatch Log Stream Deletion"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating AWS CloudWatch Log Stream Deletion
+
+Amazon CloudWatch is a monitoring and observability service that collects monitoring and operational data in the form of
+logs, metrics, and events. This data can be used to detect anomalous behavior in your environments, set alarms, visualize
+logs and metrics side by side, take automated actions, troubleshoot issues, and discover insights to keep your
+applications running smoothly.
+
+A log stream is a sequence of log events that share the same source. Each separate source of logs in CloudWatch Logs
```

Comment:
```suggestion
A log stream is a sequence of log events that share the same source. Each log source in CloudWatch Logs
```

Reply:
I'm not sure if you're talking about the log source here but wanted to make the edit in case you were.

Reply:
https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html